packages/effect/src/platform/Terminal.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @since 4.0.0
+ */
+
+import { Effect, Scope, ServiceMap } from "../index.ts"
```

Comment:
We avoid using the re-export files in library code, even if it is type only. So these should be `import * as Effect from "../Effect.ts" etc